### PR TITLE
LSOA-107 Add alert when leaving unsaved observation

### PR DESCRIPTION
**Pull request by kaniak274**

### CODE REVIEW

File: templates/observation.html [hunk 1]
```diff
@@ -414,6 +414,24 @@ <h5 class="modal-title" id="confirmModalLabel">Reset media</h5>
 
   <script>
     $(document).ready(function () {
+      var formChanged = false;
+
+      $('#form :input').on('change input', function() {
+        formChanged = true;
+      });
+
+      $('.nav-link').click(function() {
```
Comment (rkowalski4) [inline templates/observation.html]: Can't you use `beforeunload` event instead?

Comment (kaniak274) [reply templates/observation.html]: I was thinking about this but I didn't want to use it in every case (like update draft button) and I'm not sure if I can in function exclude those.

Comment (rkowalski4) [reply templates/observation.html]: As I remember you can check event target to do that, so you can save one listener with such approch.

Comment (kaniak274) [reply templates/observation.html]: I tried that but it was always set to document.

Comment (rkowalski4) [reply templates/observation.html]: so ok - we can do this as it is.

**Pull request merged**